plugin-packs/postcss-preset-env/.tape.js

```diff
@@ -20,6 +20,32 @@ module.exports = {
 			browsers: 'chrome >= 38'
 		}
 	},
+	'basic:ch88-ff78': {
+		message: 'supports { browsers: "chrome >= 88, firefox >= 78", stage: 0 } usage',
```

Comment:
Thoughts on updating this message and below to be more descriptive of what we intend? Something within these lines: 

`Modern browsers should get :is`
`Is can be disabled`,

etc

Feel like when reading these test cases it's not clear the purpose they serve.

Reply:
Updated to : 

```js
	'basic:ch88-ff78': {
		message: 'uses :is pseudo for nesting with modern browsers { browsers: "chrome >= 88, firefox >= 78", stage: 0 }',
		options: {
			browsers: 'chrome >= 88, firefox >= 78',
			stage: 0
		}
	},
	'basic:ch88-ff78:no-is-pseudo': {
		message: ':is pseudo for nesting can be disable with modern browsers  { browsers: "chrome >= 88, firefox >= 78", stage: 0, features: { nesting-rules: { noIsPseudoSelector: true } } } usage',
		options: {
			browsers: 'chrome >= 88, firefox >= 78',
			stage: 0,
			features: {
				'nesting-rules': {
					noIsPseudoSelector: true
				}
			}
		}
	},
	'basic:ch88-ff78-saf10': {
		message: 'does not use :is pseudo for nesting with an older browser { browsers: "chrome >= 88, firefox >= 78, safari >= 10", stage: 0 } usage',
		options: {
			browsers: 'chrome >= 88, firefox >= 78, safari >= 10',
			stage: 0
		}
	},
```